sdk/eventgrid/eventgrid/review/eventgrid.api.md

```diff
@@ -0,0 +1,206 @@
+## API Report File for "@azure/eventgrid"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AzureKeyCredential } from '@azure/core-auth';
+import { KeyCredential } from '@azure/core-auth';
+import { OperationOptions } from '@azure/core-http';
+import { PipelineOptions } from '@azure/core-http';
+import { RestResponse } from '@azure/core-http';
+
+export { AzureKeyCredential }
+
+// @public
+export interface CloudEvent<T> {
+    [keyof: string]: any;
+    data?: T;
+    datacontenttype?: string;
+    dataschema?: string;
+    id: string;
+    source: string;
+    subject?: string;
+    time?: Date;
+    type: string;
+}
+
+// @public
+export interface ContainerRegistryArtifactEventData {
+    action?: string;
+    id?: string;
+    target?: ContainerRegistryArtifactEventTarget;
+    timestamp?: Date;
+}
+
+// @public
+export interface ContainerRegistryArtifactEventTarget {
+    digest?: string;
+    mediaType?: string;
+    name?: string;
+    repository?: string;
+    size?: number;
+    tag?: string;
+    version?: string;
+}
+
+// @public
+export type ContainerRegistryChartDeletedEventData = ContainerRegistryArtifactEventData & {};
+
+// @public
+export type ContainerRegistryChartPushedEventData = ContainerRegistryArtifactEventData & {};
+
+// @public
+export interface ContainerRegistryEventActor {
+    name?: string;
+}
+
+// @public
+export interface ContainerRegistryEventData {
+    action?: string;
+    actor?: ContainerRegistryEventActor;
+    id?: string;
+    request?: ContainerRegistryEventRequest;
+    source?: ContainerRegistryEventSource;
+    target?: ContainerRegistryEventTarget;
+    timestamp?: Date;
+}
+
+// @public
+export interface ContainerRegistryEventRequest {
+    addr?: string;
```

Comment:
is `addr` some kind of convention in the service? I'm guessing it's a shorthand for `address` and maybe contains an IP address or a URL or something?

Reply:
> is addr some kind of convention in the service? I'm guessing it's a shorthand for address and maybe contains an IP address or a URL or something?

Yeah.  It's a value that is meaningful to the service (there should be a doc comment as well on it, but that doesn't make it's way into the API View).  The documentation for the property is [here](https://docs.microsoft.com/en-us/azure/event-grid/event-schema-container-registry#event-grid-event-schema).  I'd like to keep this name as is if possible (I expand on the rationale here a little more further down in my response to one of your comments about `EventHubCaptureFileCreatedEventData`.